src/test/java/seedu/address/storage/JsonAdaptedPersonTest.java

```diff
@@ -84,8 +92,8 @@ public void toModelType_invalidTags_throwsIllegalValueException() {
         List<JsonAdaptedTag> invalidTags = new ArrayList<>(VALID_TAGS);
         invalidTags.add(new JsonAdaptedTag(INVALID_TAG));
         JsonAdaptedPerson person =
-                new JsonAdaptedPerson(VALID_NAME, VALID_EMAIL, VALID_MODULE, invalidTags);
+                new JsonAdaptedPerson(VALID_NAME, VALID_EMAIL, VALID_MODULE,
+                        VALID_ASSIGNMENTS, invalidTags);
```

Comment:
Maybe we can add a test for invalid assignment

Reply:
Good point! Will add!